.github/workflows/apply-library-repo-templates.yml

```diff
@@ -0,0 +1,77 @@
+name: Apply Library Repo Templates and Create PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      template_type:
+        description: 'Template type'
+        default: 'Generated Connector Template (Standard)'
+        type: choice
+        options:
+          - Generated Connector Template (Standard)
+      target_repo:
+        description: 'Target repository to apply the template'
+        required: true
+        example: 'ballerina-platform/module-ballerinax-github'
```

Comment:
Do we need the GitHub organization here? It should be always `ballerina-platform`, right?

Reply:
yes. +1 for the suggestion. removed with https://github.com/ballerina-platform/ballerina-library/pull/6759/commits/c34e0cc3b808ede0a3bea44178b8589912ff4b08